### PR TITLE
[FIX] make masker report tables horizontally scrollable and not overlap with masker report images

### LINKED ISSUE #4281: tables in masker HTML report overlap with the masker image
Screenshot from 2024-02-22 09-38-39

code to reproduce

```python
from nilearn import datasets
from nilearn.maskers import NiftiLabelsMasker
from nilearn.datasets import (
    fetch_atlas_schaefer_2018,
)


def main():
    dataset = datasets.fetch_development_fmri(n_subjects=1)
    func_filename = dataset.func[0]

    atlas = fetch_atlas_schaefer_2018()

    labels = atlas.labels

    masker = NiftiLabelsMasker(
        atlas.maps,
        labels=labels,
        standardize="zscore_sample",
    )

    masker.fit_transform(func_filename)

    report = masker.generate_report()
    report.save_as_html("report.html")


if __name__ == "__main__":
    exit(main())

```

### LINKED ISSUE #4301: Tables are "unbrowsable" on narrow view ports
The table cannot be scrolled left or right

image

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes #4281 
- Closes #4301
- Extracted from #4295

Changes proposed in this pull request:

- [x] make sure tables do not overlap with the images
- [x] add horizontal scroll bar to tables

### CODE REVIEW

**Comment by Remi-Gau** [comment]
@bthirion 
if you want the diff to be easier to read we might want to get #4302 merged first, so line indent and formatting won't show on every diff for HTML and CSS.

File: doc/changes/latest.rst [hunk 1]
```diff
@@ -11,6 +11,8 @@ Fixes
 - :bdg-dark:`Code` Fix plotting of carpet plot due to a change in the coming version of matplolib (3.9.0) (:gh:`4279` by `Rémi Gau`_).
 - :bdg-dark:`Code` Fix errant warning when using ``stat_type`` in :func:`nilearn.glm.compute_contrast` (:gh:`4257` by `Eric Larson`_).
 - :bdg-dark:`Code` Fix when thresholding is applied to images by GLM reports (:gh:`4258` by `Rémi Gau`_).
+- :bdg-dark:`Code` Make masker report tables horizaontally scrollable (:gh:`4308` by `Rémi Gau`_).
```
Comment (bthirion) [inline doc/changes/latest.rst:14]:
```suggestion
- :bdg-dark:`Code` Make masker report tables horizontally scrollable (:gh:`4308` by `Rémi Gau`_).
```

**Review by bthirion — COMMENTED** [review]
Aside one small glitch, it is very hard to say anything on this. 
Can you point to some generated outputs ?

**Comment by Remi-Gau** [comment]
> Can you point to some generated outputs ?

Once #4295 is in I will ask for a full build of the doc so we can all easily check the reports.

**Comment by Remi-Gau** [comment]
Actually this DOES fix a bug I had not noticed on the MultiNiftiLabelsMaskers:

The region summary button was not displayed.

Before

image

After

image

**Comment by Remi-Gau** [comment]
# For review

### Before (current dev doc)

- maskers reports: https://nilearn.github.io/dev/modules/generated_reports/masker_reports_examples.html

### After

- maskers reports: https://output.circle-artifacts.com/output/job/488a511a-1ccb-4dc0-a988-87f3d9b5ea09/artifacts/0/dev/modules/generated_reports/masker_reports_examples.html#nifti-masker

**Comment by Remi-Gau** [comment]
Before

image

After

The parameters and regions summary buttons are below the image and thus cannot overlap with it.

image

**Comment by Remi-Gau** [comment]
### Before

Tables are not horizontally scrollabe

image

### After

First column is visible by default.

image


Tables are scrollable.

image

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Pull request merged**